python/doc/theory/reliability_sensitivity/sensitivity_sobol_from_pce.rst

```diff
@@ -20,19 +20,19 @@ As a result, Sobol' indices can be obtained analytically from the
 coefficients of the PCE (see [sudret2006]_, [sudret2008]_).
 
 Consider the input random vector
-:math:`\vect{X} = \left( X_1,\ldots,X_{n_X} \right)`
-and the output random variable :math:`Y` of the physical model:
+:math:`\inputRV = \left( X_1,\ldots,X_\inputDim \right)`
+and the output random variable :math:`Y` of the model:
 
 .. math::
-    Y = \operatorname{g}(\vect{X}).
+    Y =  \model(\inputRV).
 
 Variance and part of variance of a PCE
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Let :math:`n_X` be the dimension of the input random vector.
+Let :math:`d` be the dimension of the input random vector.
 Let :math:`P \in \Nset` be the number of coefficients in the functional
 basis.
-Let :math:`\mathcal{J}_P \subseteq \Nset^{n_X}` the set of multi-indices up
+Let :math:`\mathcal{J}_P \subseteq \Nset^{\inputDim}` the set of multi-indices up
 to the index :math:`P`.
 Depending on the way the coefficients are computed, the set of multi-indices
 is the consequence of the choice of the polynomial degree, the enumeration rule,
```

Comment:
At lines 40 and 43, 

    \tilde{h}(\bdZ)

should be replaced with

    \metaModel(\standardRV)

Reply:
ok